mopub/mopub-android-mediation
# Updated GDPR and user consent setting on Tapjoy Adapter

The Tapjoy Adapter can now update the MoPub GDPR setting to Tapjoy through fetchMoPubGDPRSettings() on network events.

How to Test
Setup
- Clone Mopub Sample app from [Mopub repo](https://github.com/mopub/mopub-android-sdk).
- Follow both Mopub-Tapjoy Adapter and Tapjoy SDK Integration setup.
-  Run Mopub Sample app and observe the GDPR values. 
     - GDPR values on mopub and Tapjoy network calls should be synchronized.
